[RPi4] /usr/bin/ld: anonymous version tag cannot be combined with other version tags

When trying to build on a Raspberry Pi 4 using Arch Linux, the follow error is encountered:

```
/usr/bin/ld: anonymous version tag cannot be combined with other version tags
```

GCC version: 12.1.0
Architecture: armv7l

The error occurs at the link stage:

```g++ -o vice_x64_libretro.so build/./vice/src/resid/dac.o build/./vice/src/resid/envelope.o build/./vice/src/resid/extfilt.o build/./vice/src/resid/filter8580new.o build/./vice/src/resid/pot.o build/./vice/src/resid/sid.o build/./vice/src/resid/version.o build/./vice/src/resid/voice.o build/./vice/src/resid/wave.o build/./vice/src/sid/resid.o build/./vice/src/sid/resid-fp.o build/./vice/src/residfp/builders/residfp-builder/residfp/version.o build/./vice/src/residfp/builders/residfp-builder/residfp/Dac.o build/./vice/src/residfp/builders/residfp-builder/residfp/EnvelopeGenerator.o build/./vice/src/residfp/builders/residfp-builder/residfp/ExternalFilter.o build/./vice/src/residfp/builders/residfp-builder/residfp/Filter.o build/./vice/src/residfp/builders/residfp-builder/residfp/Filter6581.o build/./vice/src/residfp/builders/residfp-builder/residfp/Filter8580.o build/./vice/src/residfp/builders/residfp-builder/residfp/FilterModelConfig.o build/./vice/src/residfp/builders/residfp-builder/residfp/FilterModelConfig8580.o build/./vice/src/residfp/builders/residfp-builder/residfp/Integrator.o build/./vice/src/residfp/builders/residfp-builder/residfp/Integrator8580.o build/./vice/src/residfp/builders/residfp-builder/residfp/OpAmp.o build/./vice/src/residfp/builders/residfp-builder/residfp/resample/SincResampler.o build/./vice/src/residfp/builders/residfp-builder/residfp/SID.o build/./vice/src/residfp/builders/residfp-builder/residfp/Spline.o build/./vice/src/residfp/builders/residfp-builder/residfp/WaveformCalculator.o build/./vice/src/residfp/builders/residfp-builder/residfp/WaveformGenerator.o build/./libretro/libretro-core.o build/./libretro/libretro-mapper.o build/./libretro/libretro-dc.o build/./libretro/libretro-glue.o build/./libretro/libretro-vkbd.o build/./libretro/libretro-graph.o build/./deps/libz/unzip.o build/./deps/libz/ioapi.o build/./vice/src/alarm.o build/./vice/src/attach.o build/./vice/src/autostart-prg.o build/./vice/src/autostart.o build/./vice/src/c64/c64-cmdline-options.o build/./vice/src/c64/c64-memory-hacks.o build/./vice/src/c64/c64-resources.o build/./vice/src/c64/c64.o build/./vice/src/c64/c64bus.o build/./vice/src/c64/c64cia1.o build/./vice/src/c64/c64cia2.o build/./vice/src/c64/c64cpu.o build/./vice/src/c64/c64datasette.o build/./vice/src/c64/c64drive.o build/./vice/src/c64/c64export.o build/./vice/src/c64/c64fastiec.o build/./vice/src/c64/c64gluelogic.o build/./vice/src/c64/c64iec.o build/./vice/src/c64/c64io.o build/./vice/src/c64/c64keyboard.o build/./vice/src/c64/c64mem.o build/./vice/src/c64/c64meminit.o build/./vice/src/c64/c64memlimit.o build/./vice/src/c64/c64memrom.o build/./vice/src/c64/c64memsnapshot.o build/./vice/src/c64/c64model.o build/./vice/src/c64/c64parallel.o build/./vice/src/c64/c64pla.o build/./vice/src/c64/c64printer.o build/./vice/src/c64/c64rom.o build/./vice/src/c64/c64romset.o build/./vice/src/c64/c64rsuser.o build/./vice/src/c64/c64sound.o build/./vice/src/c64/c64video.o build/./vice/src/c64/c64_256k.o build/./vice/src/c64/plus60k.o build/./vice/src/c64/plus256k.o build/./vice/src/c64/reloc65.o build/./vice/src/c64/cart/actionreplay.o build/./vice/src/c64/cart/actionreplay2.o build/./vice/src/c64/cart/actionreplay3.o build/./vice/src/c64/cart/actionreplay4.o build/./vice/src/c64/cart/atomicpower.o build/./vice/src/c64/cart/bisplus.o build/./vice/src/c64/cart/blackbox3.o build/./vice/src/c64/cart/blackbox4.o build/./vice/src/c64/cart/blackbox8.o build/./vice/src/c64/cart/blackbox9.o build/./vice/src/c64/cart/c64-generic.o build/./vice/src/c64/cart/c64-midi.o build/./vice/src/c64/cart/c64acia1.o build/./vice/src/c64/cart/c64cart.o build/./vice/src/c64/cart/c64carthooks.o build/./vice/src/c64/cart/c64cartmem.o build/./vice/src/c64/cart/c64tpi.o build/./vice/src/c64/cart/capture.o build/./vice/src/c64/cart/clockport-mp3at64.o build/./vice/src/c64/cart/clockport-rrnet.o build/./vice/src/c64/cart/clockport.o build/./vice/src/c64/cart/comal80.o build/./vice/src/c64/cart/cs8900io.o build/./vice/src/c64/cart/daa.o build/./vice/src/c64/cart/debugcart.o build/./vice/src/c64/cart/delaep7x8.o build/./vice/src/c64/cart/delaep64.o build/./vice/src/c64/cart/delaep256.o build/./vice/src/c64/cart/diashowmaker.o build/./vice/src/c64/cart/digimax.o build/./vice/src/c64/cart/dinamic.o build/./vice/src/c64/cart/drean.o build/./vice/src/c64/cart/dqbb.o build/./vice/src/c64/cart/ds12c887rtc.o build/./vice/src/c64/cart/easycalc.o build/./vice/src/c64/cart/easyflash.o build/./vice/src/c64/cart/epyxfastload.o build/./vice/src/c64/cart/ethernetcart.o build/./vice/src/c64/cart/exos.o build/./vice/src/c64/cart/expert.o build/./vice/src/c64/cart/final.o build/./vice/src/c64/cart/final3.o build/./vice/src/c64/cart/finalplus.o build/./vice/src/c64/cart/formel64.o build/./vice/src/c64/cart/freezeframe.o build/./vice/src/c64/cart/freezeframe2.o build/./vice/src/c64/cart/freezemachine.o build/./vice/src/c64/cart/funplay.o build/./vice/src/c64/cart/gamekiller.o build/./vice/src/c64/cart/georam.o build/./vice/src/c64/cart/gmod2.o build/./vice/src/c64/cart/gmod3.o build/./vice/src/c64/cart/gs.o build/./vice/src/c64/cart/ide64.o build/./vice/src/c64/cart/ieeeflash64.o build/./vice/src/c64/cart/isepic.o build/./vice/src/c64/cart/kcs.o build/./vice/src/c64/cart/kingsoft.o build/./vice/src/c64/cart/ltkernal.o build/./vice/src/c64/cart/mach5.o build/./vice/src/c64/cart/magicdesk.o build/./vice/src/c64/cart/magicformel.o build/./vice/src/c64/cart/magicvoice.o build/./vice/src/c64/cart/maxbasic.o build/./vice/src/c64/cart/mikroass.o build/./vice/src/c64/cart/multimax.o build/./vice/src/c64/cart/mmc64.o build/./vice/src/c64/cart/mmcreplay.o build/./vice/src/c64/cart/ocean.o build/./vice/src/c64/cart/pagefox.o build/./vice/src/c64/cart/partner64.o build/./vice/src/c64/cart/prophet64.o build/./vice/src/c64/cart/ramcart.o build/./vice/src/c64/cart/ramlink.o build/./vice/src/c64/cart/retroreplay.o build/./vice/src/c64/cart/reu.o build/./vice/src/c64/cart/rexep256.o build/./vice/src/c64/cart/rexutility.o build/./vice/src/c64/cart/rgcd.o build/./vice/src/c64/cart/ross.o build/./vice/src/c64/cart/rrnetmk3.o build/./vice/src/c64/cart/sdbox.o build/./vice/src/c64/cart/sfx_soundexpander.o build/./vice/src/c64/cart/sfx_soundsampler.o build/./vice/src/c64/cart/shortbus.o build/./vice/src/c64/cart/shortbus_digimax.o build/./vice/src/c64/cart/shortbus_etfe.o build/./vice/src/c64/cart/silverrock128.o build/./vice/src/c64/cart/simonsbasic.o build/./vice/src/c64/cart/snapshot64.o build/./vice/src/c64/cart/stardos.o build/./vice/src/c64/cart/stb.o build/./vice/src/c64/cart/superexplode5.o build/./vice/src/c64/cart/supergames.o build/./vice/src/c64/cart/supersnapshot.o build/./vice/src/c64/cart/supersnapshot4.o build/./vice/src/c64/cart/turtlegraphics.o build/./vice/src/c64/cart/rexramfloppy.o build/./vice/src/c64/cart/warpspeed.o build/./vice/src/c64/cart/westermann.o build/./vice/src/c64/cart/zaxxon.o build/./vice/src/c64/cart/zippcode48.o build/./vice/src/cbmdos.o build/./vice/src/cbmimage.o build/./vice/src/charset.o build/./vice/src/clipboard.o build/./vice/src/cmdline.o build/./vice/src/color.o build/./vice/src/core/ata.o build/./vice/src/core/ciacore.o build/./vice/src/core/ciatimer.o build/./vice/src/core/cs8900.o build/./vice/src/core/flash040core.o build/./vice/src/core/fmopl.o build/./vice/src/core/i8255a.o build/./vice/src/core/m93c86.o build/./vice/src/core/mc6821core.o build/./vice/src/core/riotcore.o build/./vice/src/core/scsi.o build/./vice/src/core/ser-eeprom.o build/./vice/src/core/spi-flash.o build/./vice/src/core/spi-sdcard.o build/./vice/src/core/t6721.o build/./vice/src/core/tpicore.o build/./vice/src/core/viacore.o build/./vice/src/core/rtc/bq4830y.o build/./vice/src/core/rtc/ds12c887.o build/./vice/src/core/rtc/ds1202_1302.o build/./vice/src/core/rtc/ds1216e.o build/./vice/src/core/rtc/ds1307.o build/./vice/src/core/rtc/ds1602.o build/./vice/src/core/rtc/pcf8583.o build/./vice/src/core/rtc/rtc-58321a.o build/./vice/src/core/rtc/rtc-72421.o build/./vice/src/core/rtc/rtc.o build/./vice/src/crc32.o build/./vice/src/crt.o build/./vice/src/datasette/datasette.o build/./vice/src/datasette/datasette-sound.o build/./vice/src/debug.o build/./vice/src/diag/c64_diag_586220_harness.o build/./vice/src/diskimage/diskimage.o build/./vice/src/diskimage/fsimage-check.o build/./vice/src/diskimage/fsimage-create.o build/./vice/src/diskimage/fsimage-dxx.o build/./vice/src/diskimage/fsimage-gcr.o build/./vice/src/diskimage/fsimage-p64.o build/./vice/src/diskimage/fsimage-probe.o build/./vice/src/diskimage/fsimage.o build/./vice/src/diskimage/realimage.o build/./vice/src/dma.o build/./vice/src/drive/drive-check.o build/./vice/src/drive/drive-cmdline-options.o build/./vice/src/drive/drive-resources.o build/./vice/src/drive/drive-snapshot.o build/./vice/src/drive/drive-sound.o build/./vice/src/drive/drive-writeprotect.o build/./vice/src/drive/drive.o build/./vice/src/drive/drivecpu.o build/./vice/src/drive/drivecpu65c02.o build/./vice/src/drive/driveimage.o build/./vice/src/drive/drivemem.o build/./vice/src/drive/driverom.o build/./vice/src/drive/drivesync.o build/./vice/src/drive/iec/c64exp/c64exp-cmdline-options.o build/./vice/src/drive/iec/c64exp/c64exp-resources.o build/./vice/src/drive/iec/c64exp/dolphindos3.o build/./vice/src/drive/iec/c64exp/iec-c64exp.o build/./vice/src/drive/iec/c64exp/profdos.o build/./vice/src/drive/iec/c64exp/supercard.o build/./vice/src/drive/iec/c64exp/stardos-exp.o build/./vice/src/drive/iec/cia1571d.o build/./vice/src/drive/iec/cia1581d.o build/./vice/src/drive/iec/cmdhd.o build/./vice/src/drive/iec/fdd.o build/./vice/src/drive/iec/glue1571.o build/./vice/src/drive/iec/iec-cmdline-options.o build/./vice/src/drive/iec/iec-resources.o build/./vice/src/drive/iec/iec.o build/./vice/src/drive/iec/iecrom.o build/./vice/src/drive/iec/memiec.o build/./vice/src/drive/iec/pc8477.o build/./vice/src/drive/iec/via1d1541.o build/./vice/src/drive/iec/via4000.o build/./vice/src/drive/iec/wd1770.o build/./vice/src/drive/iecieee/iecieee.o build/./vice/src/drive/iecieee/via2d.o build/./vice/src/drive/ieee/fdc.o build/./vice/src/drive/ieee/ieee-cmdline-options.o build/./vice/src/drive/ieee/ieee-resources.o build/./vice/src/drive/ieee/ieee.o build/./vice/src/drive/ieee/ieeerom.o build/./vice/src/drive/ieee/memieee.o build/./vice/src/drive/ieee/riot1d.o build/./vice/src/drive/ieee/riot2d.o build/./vice/src/drive/ieee/via1d2031.o build/./vice/src/drive/rotation.o build/./vice/src/drive/tcbm/glue1551.o build/./vice/src/drive/tcbm/mem1551.o build/./vice/src/drive/tcbm/tcbm-cmdline-options.o build/./vice/src/drive/tcbm/tcbm-resources.o build/./vice/src/drive/tcbm/tcbm.o build/./vice/src/drive/tcbm/tcbmrom.o build/./vice/src/drive/tcbm/tpid.o build/./vice/src/event.o build/./vice/src/fileio/cbmfile.o build/./vice/src/fileio/fileio.o build/./vice/src/fileio/p00.o build/./vice/src/findpath.o build/./vice/src/fliplist.o build/./vice/src/fsdevice/fsdevice-close.o build/./vice/src/fsdevice/fsdevice-cmdline-options.o build/./vice/src/fsdevice/fsdevice-filename.o build/./vice/src/fsdevice/fsdevice-flush.o build/./vice/src/fsdevice/fsdevice-open.o build/./vice/src/fsdevice/fsdevice-read.o build/./vice/src/fsdevice/fsdevice-resources.o build/./vice/src/fsdevice/fsdevice-write.o build/./vice/src/fsdevice/fsdevice.o build/./vice/src/gcr.o build/./vice/src/iecbus/iecbus.o build/./vice/src/imagecontents/diskcontents-block.o build/./vice/src/imagecontents/diskcontents-iec.o build/./vice/src/imagecontents/diskcontents.o build/./vice/src/imagecontents/imagecontents.o build/./vice/src/imagecontents/tapecontents.o build/./vice/src/init.o build/./vice/src/initcmdline.o build/./vice/src/interrupt.o build/./vice/src/joyport/bbrtc.o build/./vice/src/joyport/cardkey.o build/./vice/src/joyport/coplin_keypad.o build/./vice/src/joyport/cx21.o build/./vice/src/joyport/cx85.o build/./vice/src/joyport/inception.o build/./vice/src/joyport/joyport.o build/./vice/src/joyport/joyport_io_sim.o build/./vice/src/joyport/joystick.o build/./vice/src/joyport/lightpen.o build/./vice/src/joyport/mouse.o build/./vice/src/joyport/mouse_1351.o build/./vice/src/joyport/mouse_neos.o build/./vice/src/joyport/mouse_paddle.o build/./vice/src/joyport/mouse_quadrature.o build/./vice/src/joyport/multijoy.o build/./vice/src/joyport/ninja_snespad.o build/./vice/src/joyport/paperclip64.o build/./vice/src/joyport/protopad.o build/./vice/src/joyport/rushware_keypad.o build/./vice/src/joyport/sampler2bit.o build/./vice/src/joyport/sampler4bit.o build/./vice/src/joyport/script64_dongle.o build/./vice/src/joyport/spaceballs.o build/./vice/src/joyport/trapthem_snespad.o build/./vice/src/joyport/vizawrite64_dongle.o build/./vice/src/joyport/waasoft_dongle.o build/./vice/src/kbdbuf.o build/./vice/src/keyboard.o build/./vice/src/keymap.o build/./vice/src/lib.o build/./vice/src/lib/p64/p64.o build/./vice/src/log.o build/./vice/src/machine-bus.o build/./vice/src/machine.o build/./vice/src/main.o build/./vice/src/mainlock.o build/./vice/src/midi.o build/./vice/src/network.o build/./vice/src/opencbmlib.o build/./vice/src/palette.o build/./vice/src/parallel/parallel-trap.o build/./vice/src/parallel/parallel.o build/./vice/src/printerdrv/driver-select.o build/./vice/src/printerdrv/drv-ascii.o build/./vice/src/printerdrv/drv-raw.o build/./vice/src/printerdrv/interface-serial.o build/./vice/src/printerdrv/interface-userport.o build/./vice/src/printerdrv/output-graphics.o build/./vice/src/printerdrv/output-select.o build/./vice/src/printerdrv/output-text.o build/./vice/src/printerdrv/printer-serial.o build/./vice/src/printerdrv/printer-userport.o build/./vice/src/printerdrv/printer.o build/./vice/src/ram.o build/./vice/src/raster/raster-cache.o build/./vice/src/raster/raster-canvas.o build/./vice/src/raster/raster-changes.o build/./vice/src/raster/raster-cmdline-options.o build/./vice/src/raster/raster-line-changes-sprite.o build/./vice/src/raster/raster-line-changes.o build/./vice/src/raster/raster-line.o build/./vice/src/raster/raster-modes.o build/./vice/src/raster/raster-resources.o build/./vice/src/raster/raster-snapshot.o build/./vice/src/raster/raster-sprite-cache.o build/./vice/src/raster/raster-sprite-status.o build/./vice/src/raster/raster-sprite.o build/./vice/src/raster/raster.o build/./vice/src/rawfile.o build/./vice/src/rawnet.o build/./vice/src/resources.o build/./vice/src/romset.o build/./vice/src/rs232drv/rs232.o build/./vice/src/rs232drv/rs232drv.o build/./vice/src/rs232drv/rs232net.o build/./vice/src/rs232drv/rsuser.o build/./vice/src/screenshot.o build/./vice/src/serial/fsdrive.o build/./vice/src/serial/realdevice.o build/./vice/src/serial/serial-device.o build/./vice/src/serial/serial-iec-bus.o build/./vice/src/serial/serial-iec-device.o build/./vice/src/serial/serial-iec-lib.o build/./vice/src/serial/serial-iec.o build/./vice/src/serial/serial-realdevice.o build/./vice/src/serial/serial-trap.o build/./vice/src/serial/serial.o build/./vice/src/sid/catweaselmkiii.o build/./vice/src/sid/fastsid.o build/./vice/src/sid/hardsid.o build/./vice/src/sid/parsid.o build/./vice/src/sid/sid-cmdline-options.o build/./vice/src/sid/sid-resources.o build/./vice/src/sid/sid-snapshot.o build/./vice/src/sid/sid.o build/./vice/src/sid/ssi2001.o build/./vice/src/snapshot.o build/./vice/src/socket.o build/./vice/src/sound.o build/./vice/src/sysfile.o build/./vice/src/tape/t64.o build/./vice/src/tape/tap.o build/./vice/src/tape/tape-internal.o build/./vice/src/tape/tape-snapshot.o build/./vice/src/tape/tape.o build/./vice/src/tape/tapeimage.o build/./vice/src/tapeport/cp-clockf83.o build/./vice/src/tapeport/dtl-basic-dongle.o build/./vice/src/tapeport/sense-dongle.o build/./vice/src/tapeport/tapecart.o build/./vice/src/tapeport/tapeport.o build/./vice/src/tapeport/tape_diag_586220_harness.o build/./vice/src/traps.o build/./vice/src/userport/userport.o build/./vice/src/userport/userport_4bit_sampler.o build/./vice/src/userport/userport_8bss.o build/./vice/src/userport/userport_dac.o build/./vice/src/userport/userport_diag_586220_harness.o build/./vice/src/userport/userport_digimax.o build/./vice/src/userport/userport_io_sim.o build/./vice/src/userport/userport_joystick.o build/./vice/src/userport/userport_petscii_snespad.o build/./vice/src/userport/userport_rtc_58321a.o build/./vice/src/userport/userport_rtc_ds1307.o build/./vice/src/userport/userport_spt_joystick.o build/./vice/src/userport/userport_superpad64.o build/./vice/src/userport/userport_wic64.o build/./vice/src/util.o build/./vice/src/vdrive/vdrive-bam.o build/./vice/src/vdrive/vdrive-command.o build/./vice/src/vdrive/vdrive-dir.o build/./vice/src/vdrive/vdrive-iec.o build/./vice/src/vdrive/vdrive-internal.o build/./vice/src/vdrive/vdrive-rel.o build/./vice/src/vdrive/vdrive-snapshot.o build/./vice/src/vdrive/vdrive.o build/./vice/src/vicefeatures.o build/./vice/src/vicii/vicii-badline.o build/./vice/src/vicii/vicii-cmdline-options.o build/./vice/src/vicii/vicii-color.o build/./vice/src/vicii/vicii-draw.o build/./vice/src/vicii/vicii-fetch.o build/./vice/src/vicii/vicii-irq.o build/./vice/src/vicii/vicii-mem.o build/./vice/src/vicii/vicii-phi1.o build/./vice/src/vicii/vicii-resources.o build/./vice/src/vicii/vicii-snapshot.o build/./vice/src/vicii/vicii-sprites.o build/./vice/src/vicii/vicii-stubs.o build/./vice/src/vicii/vicii-timing.o build/./vice/src/vicii/vicii.o build/./vice/src/video/render1x1.o build/./vice/src/video/render1x1ntsc.o build/./vice/src/video/render1x1pal.o build/./vice/src/video/render1x1rgbi.o build/./vice/src/video/video-canvas.o build/./vice/src/video/video-cmdline-options.o build/./vice/src/video/video-color.o build/./vice/src/video/video-render.o build/./vice/src/video/video-render-crtmono.o build/./vice/src/video/video-render-palntsc.o build/./vice/src/video/video-render-rgbi.o build/./vice/src/video/video-resources.o build/./vice/src/video/video-sound.o build/./vice/src/video/video-viewport.o build/./vice/src/vsync.o build/./vice/src/zfile.o build/./vice/src/zipcode.o build/./vice/src/arch/shared/socketdrv/socketdrv.o build/./retrodep/embedded.o build/./retrodep/embedded/c64embedded.o build/./retrodep/info.o build/./retrodep/cart/cpmcart.o build/./retrodep/gfxoutputdrv/gfxoutput.o build/./retrodep/monitor/asm6502.o build/./retrodep/monitor/asmR65C02.o build/./retrodep/monitor/asmz80.o build/./retrodep/monitor/monitor.o build/./retrodep/monitor/monitor_network.o build/./retrodep/monitor/mon_util.o build/./retrodep/printerdrv/drv-1520.o build/./retrodep/printerdrv/drv-mps803.o build/./retrodep/printerdrv/drv-nl10.o build/./retrodep/samplerdrv/file_drv.o build/./retrodep/samplerdrv/sampler.o build/./retrodep/video/renderscale2x.o build/./retrodep/video/video-render-2x2.o build/./retrodep/archdep.o build/./retrodep/console.o build/./retrodep/joy.o build/./retrodep/lightpendrv.o build/./retrodep/kbd.o build/./retrodep/main.o build/./retrodep/mousedrv.o build/./retrodep/signals.o build/./retrodep/soundretro.o build/./retrodep/ui.o build/./retrodep/uimon.o build/./retrodep/uistatusbar.o build/./retrodep/video.o build/./retrodep/vsyncarch.o build/./deps/libz/adler32.o build/./deps/libz/crc32.o build/./deps/libz/deflate.o build/./deps/libz/gzclose.o build/./deps/libz/gzlib.o build/./deps/libz/gzread.o build/./deps/libz/gzwrite.o build/./deps/libz/inffast.o build/./deps/libz/inflate.o build/./deps/libz/inftrees.o build/./deps/libz/trees.o build/./deps/libz/zutil.o build/./deps/7zip/7zArcIn.o build/./deps/7zip/7zBuf.o build/./deps/7zip/7zCrc.o build/./deps/7zip/7zCrcOpt.o build/./deps/7zip/7zDec.o build/./deps/7zip/7zFile.o build/./deps/7zip/7zStream.o build/./deps/7zip/Bcj2.o build/./deps/7zip/Bra.o build/./deps/7zip/Bra86.o build/./deps/7zip/BraIA64.o build/./deps/7zip/CpuArch.o build/./deps/7zip/Delta.o build/./deps/7zip/Lzma2Dec.o build/./deps/7zip/LzmaDec.o build/./libretro-common/compat/compat_strl.o build/./libretro-common/compat/compat_strcasestr.o build/./libretro-common/compat/fopen_utf8.o build/./libretro-common/encodings/encoding_utf.o build/./libretro-common/file/file_path.o build/./libretro-common/file/file_path_io.o build/./libretro-common/streams/file_stream.o build/./libretro-common/streams/file_stream_transforms.o build/./libretro-common/string/stdstring.o build/./libretro-common/time/rtime.o build/./libretro-common/vfs/vfs_implementation.o -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -shared -Wl,--version-script=./libretro/link.T -Wl,--gc-sections -s -lm -fPIC -shared -Wl,--version-script=./libretro/link.T -Wl,--gc-sections -s -lm -fPIC
/usr/bin/ld: anonymous version tag cannot be combined with other version tags
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:457: vice_x64_libretro.so] Error 1
make[1]: Leaving directory '/home/pi/git/pes-packages/packages/pes-libretro-vice/src/vice-libretro'
make: *** [Makefile:444: default] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -438,10 +438,11 @@ ifeq ($(platform), theos_ios)
 	include $(THEOS_MAKE_PATH)/library.mk
 else
 
-default:
+default: info all
+
+info:
 	$(info CFLAGS: $(PLATFLAGS) $(COMMONFLAGS))
 	$(info -------)
-	$(MAKE) $(TARGET)
 
 all: $(TARGET)
 

